### ISSUE TITLE
About model training in LiTS

### ISSUE THREAD

**Issue by Seconight** [Yanfeng-Zhou/XNet#16]
Hello🥰
I tried to train the XNet3D on LiTS, the parameters have been checked against the paper, but during the training process, the accuracy of one category is relatively low compared to the other two, such as:
![image](https://github.com/Yanfeng-Zhou/XNet/assets/54622036/0d4c7f26-5a0d-4b66-900b-861a07ce2811)
(Actually, I have already experimented, but I forgot to modify the `samples.per_volume_train` in this experiment. But the output of this training is still the same phenomenon)
Is that normal?

**Comment by Seconight** [comment]
and now the results are 
<img width="177" alt="image" src="https://github.com/Yanfeng-Zhou/XNet/assets/54622036/99c93ed9-e605-4108-885a-27854fea18d4">
which are lower....

**Comment by Yanfeng-Zhou** [comment]
LiTS is a segmentation data set of liver (1) and liver tumors (2). Compared with the liver, liver tumors are a small target and have variable shapes. Tumor segmentation is relatively difficult.
You can visualize some masks to get an intuitive feel.

**Comment by Seconight** [comment]
Thank you for your reply! I got it.
But I encountered a problem in training, and the final result was `[0.7270, 0.8101, 54.06, 15.77]`, The best JC during the training process was only 0.6957. 
<img width="277" alt="image" src="https://github.com/Yanfeng-Zhou/XNet/assets/54622036/a8d8a4dd-3eca-4701-930f-c7362ab18702">
(Additionally, there is no threshold required for inference in the final output...)
Is there anything else I haven't noticed about LiTS? How do I need to make corrections?

**Comment by Yanfeng-Zhou** [comment]
There seems to be no problem with the training process so far. Since the LiTS data set is difficult to converge, I suggest you increase the epoch or reduce the learning rate, and use the training results of the last few epochs for inference.

**Comment by Seconight** [comment]
It works! Thank you very much for your reply and patience.🤗
Wishing you all the best in your research and success in all your endeavors!
